src/Zio.Tests/FileSystems/FileSystemEntryRedirect.cs

```diff
@@ -7,6 +7,7 @@
 using System.IO;
 using System.Linq;
 using Zio.FileSystems;
+using Zio.Watcher;
```

Comment:
I would like to avoid a new namespace actually and the dependencies that it brings (Zio -> Zio.Watcher, Zio.FileSystems -> Zio.Watcher...etc.)
Could you move:
- the watcher interfaces (and related events) to the top root namespace along the `IFileSystem`.
- the helper implementations (aggregate, wrap) to FileSystems